Makefile

```diff
@@ -1,11 +1,21 @@
 CC=x86_64-w64-mingw32-gcc
 
-all: truckersmp-cli.exe
+all: truckersmp-cli.exe _truckersmp-cli truckersmp-cli.bash
 
 truckersmp-cli.exe: truckersmp-cli.c
-	$(CC) $< -o $@
+	@$(CC) $< -o $@
+
+_truckersmp-cli:
+	@if command -v genzshcomp > /dev/null; then \
+		./truckersmp-cli --help | genzshcomp -f zsh > _truckersmp-cli; \
+	fi
+
+truckersmp-cli.bash:
+	@if command -v genzshcomp > /dev/null; then \
+		./truckersmp-cli --help | genzshcomp -f bash > truckersmp-cli.bash; \
+	fi
```

Comment:
Please don't suppress *all* outputs :confused:

All we want to remove is "# genzshcomp <(./truckersmp-cli --help) > _truckermp-cli", that is already removed :smiley:

Reply:
Well, now we have #107 …